### PR TITLE
Add model, factory, and migration

### PR DESCRIPTION
**Pull request by JeffreyDavidson**

### CODE REVIEW

File: database/factories/StatusFactory.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Database\Factories;
```
Comment (drewroberts) [inline database/factories/StatusFactory.php:3]: We need to add the package to this namespace: Tipoff\Statuses\Database\Factories

File: src/Models/Status.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Tipoff\Statuses\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Status extends Model
+{
+    use HasFactory;
```
Comment (drewroberts) [inline src/Models/Status.php:10]: Let's switch the HasFactory to the HasPackageFactory trait

**Review by drewroberts — COMMENTED** [review]
We should also use .stub extensions in packages on the migration files

**Comment by JeffreyDavidson** [comment]
@drewroberts As with the notes package, should  the this package contain the same HasPackageTrait from the support package?

Also I'm assuming the Status model should also extend the BaseModel class?

**Comment by drewroberts** [comment]
> @drewroberts As with the Notes package, should this package have the Status model also extend the BaseModel class?

Yes. Thank you

**Review by drewroberts — APPROVED** [review]
Thank you! 💺

**Pull request merged**